app/models/send_message.rb

```diff
@@ -0,0 +1,15 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+class SendMessage
+
+  def self.text
+    account_sid = 'AC2eb5874bc5580bf392654c9381f1b7ae'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.